mkdocs/src/doc/mkdocs.yml

```diff
@@ -51,27 +51,17 @@ markdown_extensions:
             class: mermaid
             format: !!python/name:pymdownx.superfences.fence_code_format
 
-# Dev server binding
-#dev_addr: 127.0.0.1:3001
+# Dev server binding for livereload
+dev_addr: 127.0.0.1:3001
 
 nav:
   - Home:
       - index.md
-      - User guides: user_guide/introduction.md
+      - User guides: https://kotlinlang.org/docs/dokka-introduction.html
       - Developer guides: developer_guide/introduction.md
       - Community: community/slack.md
-      - FAQ: faq.md
   - User guides:
-      - User guides: user_guide/introduction.md
-      - Applying Dokka:
-          - Gradle: user_guide/applying/gradle.md
-          - Maven: user_guide/applying/maven.md
-          - Command line: user_guide/applying/cli.md
-      - Output formats:
-          - HTML: user_guide/output-formats/html.md
-      - Plugins:
-          - Versioning plugin: user_guide/plugins/versioning-plugin.md
-          - Android plugin: user_guide/plugins/android-plugin.md
+      - https://kotlinlang.org/docs/dokka-introduction.html
```

Comment:
There will still be the "User guides" button in the header, but it will lead to the documentation on kotlinlang